_includes/codepens/full-featured/index.js

```diff
@@ -235,6 +235,7 @@ tinymce.init({
   },
   menubar: 'file edit view insert format tools table tc help',
   toolbar: 'undo redo | bold italic underline strikethrough | fontselect fontsizeselect formatselect | alignleft aligncenter alignright alignjustify | outdent indent |  numlist bullist checklist | forecolor backcolor casechange permanentpen formatpainter removeformat | pagebreak | charmap emoticons | fullscreen  preview save print | insertfile image media pageembed template link anchor codesample | a11ycheck ltr rtl | showcomments addcomment',
+  toolbar_sticky: true,
```

Comment:
I'm not sure we actually should turn this on here, or at least at the moment. Reason being the website CSS currently breaks this since the website header has a ridiculously high z-index and will always force the editor to be below it. As such you wouldn't be able to get a good demo of this.

Reply:
leaving on in case we change the site. I'll add a sticky toolbar demo on an external site (codepen or fiddle)